### ISSUE TITLE
API_URL does not work for v3.19.5

### ISSUE THREAD

**Issue by Behoston** [swagger-api/swagger-ui#5005]
```yaml
  swagger-ui:
    image: swaggerapi/swagger-ui:v3.19.5
    environment:
      - API_URL=/api/swagger.json
    networks:
      - default

```
![obraz](https://user-images.githubusercontent.com/7823689/48032203-02317480-e157-11e8-9c5f-0f441911c254.png)
works fine for v3.19.4

**Comment by shockey** [comment]
Yep - bear with us, there's a small bug with Docker environment injection. This will be fixed today/tomorrow.

**Comment by shockey** [comment]
This is now live: our `latest` and `v3.19.5` images have been updated with the fixes from #5006 and #5007.

**Comment by shockey** [comment]
Thanks for reporting this, @Behoston!
